skimage/transform/_thin_plate_splines.py

```diff
@@ -0,0 +1,325 @@
+import numpy as np
+import scipy as sp
+
+
+class TpsTransform:
+    """Thin plate splines transformation.
+
+    Apply thin plates spline transformation between a set of control points.
+    It interpolates a surface that passes through each control point.
+    Control points are position constraints on a bending surface. The ideal
+    surface is one that bends the least.
+
+    Attributes
+    ----------
+    parameters : (N, D) array_like
+        Coefficients for every control points.
+    src : (N, 2) array_like
+        Control point at source coordinates
+
+    Examples
+    --------
+    >>> import skimage as ski
+
+    Define source and destination points and generate meshgrid for transformation:
+
+    >>> src = np.array([[0, 0], [0, 5], [5, 5],[5, 0]])
+    >>> dst = np.roll(src, 1, axis=0)
+    >>> coords = np.meshgrid(np.arange(5), np.arange(5))
+    >>> t_coords = np.vstack([coords[0].ravel(), coords[1].ravel()]).T
+
+    >>> tps = ski.transform.TpsTransform()
+    >>> tps.estimate(src, dst)
+    True
+
+    Apply the transformation
```

Comment:
```suggestion
    Apply the transformation:
```
(just to be consistent with the above)